src/sources/dataModel/taxonomic-service.json

```diff
@@ -210,52 +429,104 @@
                     "examples": [
                         "cetaf.org"
                     ]
                 }
             },
             "additionalProperties": false
         },
-        "schema:Maintainer": {
+        "schema:Author": {
             "type": "array",
-            "description": "An array of maintainers of a Dataset, software package (SoftwareApplication), or other Project. A maintainer is a Person or Organization that manages contributions to, and/or publication of, some (typically complex) artefact.",
+            "description": "An array of authors of this service.",
             "items": {
                 "type": "object",
                 "properties": {
                     "schema:identifier": {
+                        "@id": "orcid",
+                        "@type": "PropertyValue",
+                        "propertyID": "https://registry.identifiers.org/registry/orcid",
                         "type": "string",
-                        "description": "A unique identifier to identify the maintainer; ORCID or Wikidata identifiers are valid",
+                        "description": "A unique identifier to identify the author; ORCID identifiers are valid",
                         "examples": [
-                            "https://orcid.org/0000-0000-0000-0000"
+                            "https://api.github.com/users/username"
                         ]
                     },
+                    "schema:affiliation": {
+                        "type": "object",
+                        "properties": {
+                            "schema:identifier": {
+                                "type": "string",
+                                "description": "A unique identifier to identify the author's affiliation; ROR identifiers are valid",
+                                "examples": [
+                                    "https://ror.org/0566bfb96"
+                                ]
+                            },
+                            "schema:name": {
+                                "type": "string",
+                                "description": "Full name of the author's affiliated orgnisation",
+                                "examples": [
+                                    "Naturalis Biodiversity Center"
+                                ]
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "schema:Maintainer": {
+            "type": "array",
+            "description": "An array of maintainers of a Dataset, software package (SoftwareApplication), or other Project. A maintainer is a Person or Organization that manages contributions to, and/or publication of, some (typically complex) artefact.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "schema:identifier": [
+                        {
+                            "@id": "github",
```

Comment:
@id and @type and propertyID are not valid json schema properties. If you require this to be a github ID then you could enforce it with a pattern

Reply:
Got that from an example somewhere, but indeed is not schema, removed those properties just to leave: type, description and examples